[icons] docs: add many tags, word associations to icons metadata

gabrieljroth opened the pull request:
#### Fixes #0000

#### Checklist

- [ ] Includes tests
- [ ] Update documentation

<!-- DO NOT enable CircleCI for your fork. Our build will run when you open this PR. -->

#### Changes proposed in this pull request:

Added tags to each icon.

#### Reviewers should focus on:

Tags were added to the file in the correct manner.

#### Screenshot

<!-- Include an image of the most relevant user-facing change, if any. -->

Commit:
enriched icons.json with hack week icon word association results

palantirtech commented:
Thanks for your interest in [palantir/blueprint](https://github.com/palantir/blueprint), @gabrieljroth! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Blank",
         "iconName": "blank",
-        "tags": "empty, placeholder",
+        "tags": "[empty, placeholder, nothing, blank]",
```
we don't need the square brackets in this field, please remove those across the whole file. it should be easy to do a find/replace on `"[` -> `"` and `]"` -> `"` in this file

adidahiya replied:
also, the icon name ("blank" in this case) doesn't need to appear in the tags list. the search algorithm includes that name when searching for icons already

gabrieljroth replied:
Good catches! Both changes have been made.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Style",
         "iconName": "style",
-        "tags": "visual style, editor",
+        "tags": "[visual style, editor, paint, paintbrush, color, add color, compose, edit, create new, draw, canvas]",
```
these are mostly good but I'm not convinced about "edit" and "create new"... we already have an "edit" icon and "create new" doesn't feel like it applies

gabrieljroth replied:
removed them

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Align: left",
         "iconName": "align-left",
-        "tags": "text flow, alignment, justification, range, flush left",
+        "tags": "[text flow, alignment, justification, range, flush left, align left, left align, rows, left aligned, document]",
```
again, we don't need "align left", "left align", or "left aligned", since those are covered by the icon name already

gabrieljroth replied:
removed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Align: right",
         "iconName": "align-right",
-        "tags": "text flow, alignment, justification, range, flush right",
+        "tags": "[text flow, alignment, justification, range, flush right, right align, align, text align]",
```
none of these added tags change the search algorithm, so let's prefer brevity and omit them

gabrieljroth replied:
removed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Align: center",
         "iconName": "align-center",
-        "tags": "text flow, alignment, justification, range, centered",
+        "tags": "[text flow, alignment, justification, range, centered, text, lines, filing]",
```
what does "filing" mean in this context?

gabrieljroth replied:
removed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Align: justify",
         "iconName": "align-justify",
-        "tags": "text flow, alignment, justification, range, justified",
+        "tags": "[text flow, alignment, justification, range, justified, lines, menu , stack , stack, settings, list]",
```
I think these new tags actually hurt the existing icon semantics. `menu` is its own icon, and conflating `align-justify` with a "menu" could lead people choose the wrong icon. I don't think we should add these tags just because the two icons are visually similar.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Underline",
         "iconName": "underline",
-        "tags": "typography, text, font style, underline, underscore",
+        "tags": "[typography, text, font style, underline, underscore, a]",
```
I think "anchor" and "link" would be more appropriate. Searching with the query "a" is meaningless on the icons page.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Search around",
         "iconName": "search-around",
-        "tags": "search, exploration, information, area, graph",
+        "tags": "[search, exploration, information, area, graph, search around, link, connection, dot, network, links, connections, connect]",
```
we don't need "search around" or "search" here but the other additions are good

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Layout: auto",
         "iconName": "layout-auto",
-        "tags": "layout, presentation, arrangement, auto, graph, grid",
+        "tags": "[layout, presentation, arrangement, auto, graph, grid, covid, link, network, snowflake, atom, connection]",
```
"covid" is inappropriate here since we already have a "virus" icon. but the other additions are good 👍🏽

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Group objects",
         "iconName": "group-objects",
-        "tags": "group, alignment, organization, arrangement, classification, objects",
+        "tags": "[group, alignment, organization, arrangement, classification, objects, toggle, toggle button, radio buttons, lights, poll, pig, radio button, switch, two dots, plug, snout, adapter]",
```
I don't think this icon really looks like a toggle button, radio button, or switch... we should design new icons for those if we need to represent them. not a big fan of "poll" either -- a "radio-button" icon ought to cover that use case. "lights", "pig", "snout", "two dots", and "plug" are ok.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Merge into links",
         "iconName": "merge-links",
-        "tags": "merge, combine, consolidate, jointment, links",
+        "tags": "[merge, combine, consolidate, jointment, links, switch, button, toggle, kayak, seating, battle droid, outlet, connector]",
```
same here, I don't think switch or button are appropriate... and "kayak" / "battle droid" feel like a stretch :)

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Layout: grid",
         "iconName": "layout-grid",
-        "tags": "layout, presentation, arrangement, grid, graph, grid",
+        "tags": "[layout, presentation, arrangement, grid, graph, grid, module, app, arrange]",
```
"app" feels too generic, let's drop that one. the others are good 👍🏽

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Layout: skew grid",
         "iconName": "layout-skew-grid",
-        "tags": "layout, presentation, arrangement, skew, graph, grid",
+        "tags": "[layout, presentation, arrangement, skew, graph, grid, eight, dots, nodes, pot holes]",
```
"eight" and "pot holes" don't feel appropriate, can you justify those additions?

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Heatmap",
         "iconName": "heatmap",
-        "tags": "hierarchy, matrix, heat map",
+        "tags": "[hierarchy, matrix, heat map, bubbles, circles, graph, splash, blob, heatmap, ideas]",
```
don't need "heatmap" here, and "ideas" feels like a stretch

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Select",
         "iconName": "select",
-        "tags": "selection, arrow, cursor, area, range",
+        "tags": "[selection, arrow, cursor, area, range, draw, drag, select, click, open, expand, minimize, manipulate, check box, crop]",
```
don't need "select" here. "open" and "check box" feel like a stretch

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Predictive analysis",
         "iconName": "predictive-analysis",
-        "tags": "analysis, investigation, search, study, test, brain",
+        "tags": "[analysis, investigation, search, study, test, brain, it\u2019s a brain, definitely a brain, could mean anything, but, yep, still a brain]",
```
I don't think you meant to add this, please rever this line

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Locate",
         "iconName": "locate",
-        "tags": "target, location, destination, mark, map, area",
+        "tags": "[target, location, destination, mark, map, area, zoom, identify, locate, crosshairs]",
```
we have other zoom icons, so I don't think "zoom" is appropriate here

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Line chart",
         "iconName": "timeline-line-chart",
-        "tags": "graph, line, chart",
+        "tags": "[graph, line, chart, line chart, upward trend, time series, line graph]",
```
don't need "line chart" or "line graph" here, they're redundant

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Bar chart",
         "iconName": "timeline-bar-chart",
-        "tags": "graph, bar, chart",
+        "tags": "[graph, bar, chart, horizontal histogram, bar chart]",
```
we don't need "bar chart" here, it's redundant. also, did you mean "vertical histogram" here?

<img width="151" alt="image" src="https://user-images.githubusercontent.com/723999/179644806-09f8df98-87ba-4045-aa35-d2ba7bf193df.png">

gabrieljroth replied:
Some refer to that as "horizontal" because the axis of the bases of the bars is horizontal. But on the other hand the bars themselves are vertical. I'll remove horizontal for simplicity--"histogram" should do.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Applications",
         "iconName": "applications",
-        "tags": "application, browser, windows, platforms",
+        "tags": "[application, browser, windows, platforms, pages, duplicate, copy, page, window, website, files]",
```
we already have "windows", we can drop one of them (probably keep "window"). I don't think conflating this icon with "pages" or "files" will be good in the long run... "webpage" might be ok though.

gabrieljroth replied:
I don't know if I agree. The reason I crowdsourced this icon tagging was because people have a hard time finding icons. I think accepting people's loose associations will make searching a lot more powerful. If this icon comes up when someone searched "files" or "pages" I don't think it would be an issue. They'll have different options and they can choose one. But if you disagree I'm happy to remove them!

adidahiya replied:
let's keep "page" (with the intention that it's an abbreviation of "webpage"), add "webpage" too, and leave out "files". kind of a middle ground.

gabrieljroth replied:
done!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Projects",
         "iconName": "projects",
-        "tags": "drawer, sections",
+        "tags": "[drawer, sections, files, open, directory, folder, ebulliant, briefcase, filing]",
```
"open", "ebulliant" (not a word?), and "briefcase" don't feel appropriate here

gabrieljroth replied:
I agree on "ebullient" (maybe misspelled) and "briefcase" but i think "open" is a great tag--the icon sort of looks like opening a file drawer! No?

adidahiya replied:
alright, you can keep "open"

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Notifications",
         "iconName": "notifications",
-        "tags": "notifications, bell, alarm, notice, warning",
+        "tags": "[notifications, bell, alarm, notice, warning, notification, alert]",
```
"notification" is redundant, but "alert" is good

gabrieljroth replied:
removed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2,3899 +2,3899 @@
         "displayName": "Unlock",
         "iconName": "unlock",
-        "tags": "unlock, disengage, disconnect, separate, open",
+        "tags": "[unlock, disengage, disconnect, separate, open, unlocked, plaintext, authenticated, open lock, unsafe, editing]",
```
"editing" feels like a stretch

gabrieljroth replied:
I think that association was made because that icon is used to unlock when you want to edit something (like in Mac system preferences). I could see someone making an Edit button that requires certain credentials and wanting to use that icon. Let me know if you still want me to remove it though.

adidahiya replied:
alright, you can keep it

adidahiya reviewed (changes requested):
Hey @gabrieljroth, I haven't forgotten about this PR, it's just very large and will take a while to review. I've looked through the first 300 lines and this is my feedback so far, but I'm sure I will have more feedback for the rest of the changes. If you want to merge these changes sooner, you can break this down into smaller PRs. Otherwise, you can wait until I'm able to review the whole thing here.

Commit:
removed single quotations from tags and removed tags that equal iconName

Commit:
removed tags as per Adi Dahiya's comments

gabrieljroth commented:
@adidahiya, thanks for reviewing the first chunk of the updates! I removed the quotations, the brackets, and any tag that was redundant with the Icon name. I also removed specific tags as per your comments (except ones where I disagreed per my replies).

Commit:
removed tags as per Adi Dahiya's comments

adidahiya commented on packages/icons/icons.json:
```diff
@@ -303,63 +303,63 @@
         "displayName": "More",
         "iconName": "more",
-        "tags": "dots, three, extra, new, options",
+        "tags": "dots, three, extra, new, options, ellipses, break, pause, elipse, dot dot dot, suspense, see more, loading",
```
```suggestion
        "tags": "dots, three, extra, new, options, ellipses, break, pause, suspense, see more, loading",
```

- not sure about spelling of "elipse" / its relation to the mathematical shape
- "dot dot dot" is basically redundant with "dot"

adidahiya commented on packages/icons/icons.json:
```diff
@@ -303,63 +303,63 @@
         "displayName": "Media",
         "iconName": "media",
-        "tags": "audio, video, media, picture, image, drawing, illustration",
+        "tags": "audio, video, picture, image, drawing, illustration, mountain, photo",
```
```suggestion
        "tags": "audio, video, picture, image, drawing, illustration, landscape, photo",
```

I think "landscape" is more appropriate than "mountain"

gabrieljroth replied:
I think both are good tbh

adidahiya commented on packages/icons/icons.json:
```diff
@@ -303,63 +303,63 @@
         "displayName": "Link",
         "iconName": "link",
-        "tags": "link, connection, network",
+        "tags": "connection, network, paperclip, attach",
```
```suggestion
        "tags": "connection, network, chain, attach",
```

it's not a paperclip, it's a link in a chain

adidahiya commented on packages/icons/icons.json:
```diff
@@ -303,63 +303,63 @@
         "displayName": "Share",
         "iconName": "share",
-        "tags": "share, square, arrow",
+        "tags": "square, arrow, export, maximize, enlarge, fit screen, open new tab, open new page, link, open new window, open, new tab",
```
```suggestion
        "tags": "square, arrow, export, open, new page, link, new window, new tab",
```

we already have a "maximize" icon, I don't want to conflate them

adidahiya commented on packages/icons/icons.json:
```diff
@@ -303,63 +303,63 @@
         "displayName": "Download",
         "iconName": "download",
-        "tags": "circle, arrow, down, downloading",
+        "tags": "circle, arrow, down, downloading, down arrow",
```
```suggestion
        "tags": "circle, arrow, down, downloading",
```

we already have "down" and "arrow" as tags here

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Import",
         "iconName": "import",
-        "tags": "arrow, down, importing,",
+        "tags": "arrow, down, importing, , download",
```
```suggestion
        "tags": "arrow, down, importing, download",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Maximize",
         "iconName": "maximize",
-        "tags": "arrows, increase, bigger",
+        "tags": "arrows, increase, bigger, expand, opposite diagonal arrows, diagonal bidirectional arrows, double diagonal arrows",
```
```suggestion
        "tags": "arrows, increase, bigger, expand, opposite, diagonal, bidirectional",
```

just simplifying a bit

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Tick",
         "iconName": "tick",
-        "tags": "mark, sign, ok, approved, success",
+        "tags": "mark, sign, ok, approved, success, check, verizon, done, complete, confirm",
```
```suggestion
        "tags": "mark, sign, ok, approved, success, check, done, complete, confirm",
```

not a fan of the brand name addition, but the other ones are good

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Cross",
         "iconName": "cross",
-        "tags": "cross mark, fail, delete, no, close, remove",
+        "tags": "cross mark, fail, delete, no, close, remove, not available, wrong/incorrect, cancel, x",
```
```suggestion
        "tags": "cross mark, fail, delete, no, close, remove, not, available, unavailable, wrong, incorrect, cancel",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Plus",
         "iconName": "plus",
-        "tags": "sign, add, maximize, zoom in",
+        "tags": "sign, add, maximize, zoom in, add or append, enlarge or zoom-in, increase",
```
```suggestion
        "tags": "sign, add, maximize, zoom in, append, enlarge, increase",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Arrow: left",
         "iconName": "arrow-left",
-        "tags": "arrow, direction, left",
+        "tags": "arrow, direction, left, left arrow, backspace, undo, back",
```
```suggestion
        "tags": "arrow, direction, left, backspace, back",
```

removed a redundant tag. also we already have an "undo" icon, I don't want to conflate them

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "Arrow: right",
         "iconName": "arrow-right",
-        "tags": "arrow, direction, right",
+        "tags": "arrow, direction, right, next, right arrow",
```
```suggestion
        "tags": "arrow, direction, right, next",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -373,91 +373,91 @@
         "displayName": "List",
         "iconName": "list",
-        "tags": "agenda, four lines, table",
+        "tags": "agenda, four lines, table, menu, rows, hamburger, document, writing, justified, hamburger menu, block text, horizontal bars",
```
I don't want to conflate this with "menu", we clearly have two separate icons for lists vs. menus. Same with "align-justify".

```suggestion
        "tags": "agenda, four lines, table, rows, document, writing, block text, horizontal bars",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Zoom out",
         "iconName": "zoom-out",
-        "tags": "search, magnifying glass, minus",
+        "tags": "search, magnifying glass, minus, remove",
```
not a big fan of this, there are many other icons which are better suited for "removal"

gabrieljroth replied:
Fair enough

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Zoom in",
         "iconName": "zoom-in",
-        "tags": "search, magnifying glass, plus",
+        "tags": "search, magnifying glass, plus, find, new, zoom, magnify, enlarge",
```
"new" feels unnecessary here

gabrieljroth replied:
I think it's fine. The tag probably came from the plus in the icon. I think worth keeping it having in mind the icon searching problem. But if you feel strongly you can remove it!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Cog",
         "iconName": "cog",
-        "tags": "settings, circle,",
+        "tags": "settings, circle, , wheel, mechanical, tools, gear",
```
```suggestion
        "tags": "settings, circle, wheel, mechanical, tools, gear",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Flag",
         "iconName": "flag",
-        "tags": "map, position, country, nationality",
+        "tags": "map, position, country, nationality, flagpole, flag pole",
```
```suggestion
        "tags": "map, position, country, nationality, pole",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Error",
         "iconName": "error",
-        "tags": "notification, failure, circle, exclamation mark, sign",
+        "tags": "notification, failure, circle, exclamation mark, sign, contour, warning, attention, look out",
```
```suggestion
        "tags": "notification, failure, circle, exclamation mark, sign, warning, attention",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Credit card",
         "iconName": "credit-card",
-        "tags": "payment, bank, transaction",
+        "tags": "payment, bank, transaction, webpage, document, \"i dont know",
```
```suggestion
        "tags": "payment, bank, transaction",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Logout",
         "iconName": "log-out",
-        "tags": "arrow, leave",
+        "tags": "arrow, leave, next, share, open in new tab, exit",
```
we shouldn't conflate this with the real "share" icon

```suggestion
        "tags": "arrow, leave, next, exit, sign out, auth, authentication",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Star: empty",
         "iconName": "star-empty",
-        "tags": "shape, unpin, mark",
+        "tags": "shape, unpin, mark, star, favorite, like, empty star, goal, accomplishment",
```
```suggestion
        "tags": "shape, unpin, mark, star, favorite, like, goal, accomplishment",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Sort: alphabetical",
         "iconName": "sort-alphabetical",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, sort, alphabetize, sort ascending, sort descending, string sort, sort alphabetically",
```
```suggestion
        "tags": "ascending, array, arrange, sort, alphabetize, alphabetical, string, compare",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Sort: numerical",
         "iconName": "sort-numerical",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, asc, sort, numbers, dropdown, range",
```
```suggestion
        "tags": "ascending, array, arrange, sort, numbers, range",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Sort",
         "iconName": "sort",
-        "tags": "ascending, array, arrange",
+        "tags": "ascending, array, arrange, descending list, sort list, scroll down, menu, scroll, hamburger, descending, decrease, justified",
```
```suggestion
        "tags": "ascending, array, arrange, sort, list, descending, scroll down, menu, scroll, decrease",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Caret: up",
         "iconName": "caret-up",
-        "tags": "direction, order, up",
+        "tags": "direction, order, up, chevron, arrow, back, pointer left, position marker",
```
```suggestion
        "tags": "direction, order, up, chevron, triangle, arrow",
```

gabrieljroth replied:
Point seems good in theory but if we don't have that tag for all arrow-like icons it would be inconsistent to have it here.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Caret: right",
         "iconName": "caret-right",
-        "tags": "direction, order, right",
+        "tags": "direction, order, right, play, run, execute, triangle, arrow, arrow right, triangle right",
```
I don't want to conflate this with "play"

```suggestion
        "tags": "direction, order, right, triangle, arrow",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Caret: down",
         "iconName": "caret-down",
-        "tags": "direction, order, down",
+        "tags": "direction, order, down, down arrow, arrow, solid triangle down",
```
```suggestion
        "tags": "direction, order, down, triangle, arrow",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Menu: opened",
         "iconName": "menu-open",
-        "tags": "show, navigation",
+        "tags": "show, navigation, menu right, menu, open menu, right, expand, indent",
```
```suggestion
        "tags": "show, navigation, menu, open, right, expand, indent",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Menu: closed",
         "iconName": "menu-closed",
-        "tags": "hide, navigation",
+        "tags": "hide, navigation, list",
```
```suggestion
        "tags": "hide, navigation, list, left, dedent",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "Feed",
         "iconName": "feed",
-        "tags": "rss, feed",
+        "tags": "rss, wifi, signal, radio, network, rss feed",
```
```suggestion
        "tags": "rss, wifi, signal, radio, network",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -471,280 +471,280 @@
         "displayName": "One column",
         "iconName": "one-column",
-        "tags": "layout, columns, switch, change, one",
+        "tags": "layout, columns, switch, change, one, left, exit, minimiz, eject wherever, wall moves this way, collapse left",
```
```suggestion
        "tags": "layout, columns, switch, change, one, left, exit, collapse",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "Disable",
         "iconName": "disable",
-        "tags": "off, circle, remove",
+        "tags": "off, circle, remove, not, do not enter, no, anti, cancel, exit, no selection, do not",
```
```suggestion
        "tags": "off, circle, remove, do not enter, no, anti, cancel, exit, no selection",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "Unpin",
         "iconName": "unpin",
-        "tags": "map, position, safety pin, detach",
+        "tags": "map, position, safety pin, detach, branch, git, version, pushpin, pin, thumbtack",
```
```suggestion
        "tags": "map, position, safety pin, detach, version, pushpin, pin, thumbtack",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "New link",
         "iconName": "new-link",
-        "tags": "create, add, plus, links",
+        "tags": "create, add, plus, links, add link, add to chain, insert, addevent",
```
```suggestion
        "tags": "create, add, plus, link, chain, insert",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "New object",
         "iconName": "new-object",
-        "tags": "create, add, plus, objects, circle",
+        "tags": "create, add, plus, objects, circle, add object, add element, new entity",
```
```suggestion
        "tags": "create, add, plus, objects, circle, element, entity",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "Path search",
         "iconName": "path-search",
-        "tags": "map, magnifying glass, position, location",
+        "tags": "map, magnifying glass, position, location, search, map search, locate",
```
some of these are redundant

```suggestion
        "tags": "map, magnifying glass, position, location, locate",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "Code",
         "iconName": "code",
-        "tags": "code, markup, language, tag",
+        "tags": "markup, language, tag, idk lol",
```
```suggestion
        "tags": "markup, language, tag",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -772,84 +772,84 @@
         "displayName": "Search text",
         "iconName": "search-text",
-        "tags": "magnifying glass, exploration",
+        "tags": "magnifying glass, exploration, see text, text, text zoom, search, finder, examine, text search",
```
```suggestion
        "tags": "magnifying glass, exploration, see, zoom, finder, examine",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -863,14 +863,14 @@
         "displayName": "Paperclip",
         "iconName": "paperclip",
-        "tags": "attachments, add",
+        "tags": "attachments, add, paper clip, attach, attachment, clippy",
```
```suggestion
        "tags": "attachment, add, paper clip, attach, clippy",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -863,14 +863,14 @@
         "displayName": "Label",
         "iconName": "label",
-        "tags": "text, tag, ticket",
+        "tags": "text, tag, ticket, file, document, mail, print, paper, edit",
```
```suggestion
        "tags": "text, tag, ticket, file, document, mail, print, paper",
```

adidahiya reviewed (changes requested):
reviewed up to line 900

adidahiya commented on packages/icons/icons.json:
```diff
@@ -891,21 +891,21 @@
         "displayName": "Table: list",
         "iconName": "th-list",
-        "tags": "index, rows, list, order, series",
+        "tags": "index, rows, list, order, series, calendar, table",
```
```suggestion
        "tags": "index, rows, list, order, series, table",
```

we have a strictly better icon for "calendar", let's not conflate them

adidahiya commented on packages/icons/icons.json:
```diff
@@ -891,21 +891,21 @@
         "displayName": "Table: derived",
         "iconName": "th-derived",
-        "tags": "get, obtain, take, acquire, index, rows, columns, list",
+        "tags": "get, obtain, take, acquire, index, rows, columns, list, monocle, table row, row, table, add to dataset, go to dataset",
```
```suggestion
        "tags": "get, obtain, take, acquire, index, rows, columns, list, table, row, add to dataset, go to dataset, provenance",
```

I don't think the monocle product name should leak into blueprint, and it might get renamed down the line anyway

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Send to",
         "iconName": "send-to",
-        "tags": "circle, export, arrow",
+        "tags": "circle, export, arrow, outside, external, pop out, open object, open item, ellipses, array",
```
```suggestion
        "tags": "circle, export, arrow, outside, external, pop out, open object, open item",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Eye",
         "iconName": "eye-open",
-        "tags": "show, visible, clear, view, vision",
+        "tags": "show, visible, clear, view, vision, eye, see, eye and check",
```
```suggestion
        "tags": "show, visible, clear, view, vision, see",
```

there's no "check" here. and "eye" is redundant

gabrieljroth replied:
I think the idea behind "check" is that this icon can be used to denote a review or check action (something needs eyes on it)

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Arrow: up",
         "iconName": "arrow-up",
-        "tags": "direction, north",
+        "tags": "direction, north, up arrow, arrow, up, upwards",
```
```suggestion
        "tags": "direction, north, upwards",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Arrow: down",
         "iconName": "arrow-down",
-        "tags": "direction, south",
+        "tags": "direction, south, down, down arrow, please see below, descend, scroll down",
```
```suggestion
        "tags": "direction, south, see below, descend, scroll down",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Arrows: horizontal",
         "iconName": "arrows-horizontal",
-        "tags": "direction, level",
+        "tags": "direction, level, arrows, bidirectional, left-right, resize, move horizontally, swap, arrow, switch, link, reaction, alternate",
```
```suggestion
        "tags": "direction, level, bidirectional, left-right, resize, move horizontally, swap, switch, link, reaction, alternate",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Arrows: vertical",
         "iconName": "arrows-vertical",
-        "tags": "direction, level",
+        "tags": "direction, level, arrows, two way, up down, drag up or down, expand, collaps, open, up and down, up down arrow",
```
```suggestion
        "tags": "direction, level, two way, up down, drag up or down, expand, collapse, open",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Resolve",
         "iconName": "resolve",
-        "tags": "circles, divide, split",
+        "tags": "circles, divide, split, intersection, overlap, venn diagram",
```
💯

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Briefcase",
         "iconName": "briefcase",
-        "tags": "suitcase, business, case, baggage,",
+        "tags": "suitcase, business, case, baggage, , work, kit, toolbox",
```
```suggestion
        "tags": "suitcase, business, case, baggage, work, kit, toolbox",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Delta",
         "iconName": "delta",
-        "tags": "alt j, symbol",
+        "tags": "alt j, symbol, triangle, diff",
```
```suggestion
        "tags": "symbol, triangle, diff, change, formula, greek, alphabet",
```

while we're here, we should really remove "alt j" 🤦🏽

gabrieljroth replied:
haha that's hilarious

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Barcode",
         "iconName": "barcode",
-        "tags": "product, scan,",
+        "tags": "product, scan, ",
```
```suggestion
        "tags": "product, scan, identifier, code",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Widget",
         "iconName": "widget",
-        "tags": "square, corners",
+        "tags": "square, corners, text box, fence, highlight area, select area, the \"draw a rectangle tool\" in preview, sqaure, grid, connection",
```
```suggestion
        "tags": "square, corners, text box, fence, highlight area, select area, draw rectangle, grid, connection",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Unresolve",
         "iconName": "unresolve",
-        "tags": "split, divide, disconnect, separate",
+        "tags": "split, divide, disconnect, separate, sets, substraction, pancakes, union, group, join, venn diagram, overlap, circles",
```
```suggestion
        "tags": "split, divide, disconnect, separate, sets, subtraction, pancakes, union, group, join, venn diagram, overlap, circles",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Add to artifact",
         "iconName": "add-to-artifact",
-        "tags": "list, plus",
+        "tags": "list, plus, text, add, create, add item, add to list, menu, more, hamburger, add section, add page",
```
```suggestion
        "tags": "list, plus, text, add, create, add item, add to list, add section, add page",
```

gabrieljroth replied:
Somebody likes hamburgers

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Chart",
         "iconName": "chart",
-        "tags": "arrow, increase, up, line, bar, graph",
+        "tags": "arrow, increase, up, line, bar, graph, trend, growth, ?, stonks, increasing trend, trendline, upward",
```
```suggestion
        "tags": "arrow, increase, up, line, bar, graph, trend, growth, increasing trend, trendline, upward",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Direction: left",
         "iconName": "direction-left",
-        "tags": "pointer, west",
+        "tags": "pointer, west, direction, space, arrow left, back, pointer left",
```
what does "space" mean here?

gabrieljroth replied:
maybe it looks like a spacecraft?

adidahiya replied:
ping on this

gabrieljroth replied:
going to delete "space" and add "plane" and "aircraft" which I think are good tags (for both direction-left and direction-right)

adidahiya commented on packages/icons/icons.json:
```diff
@@ -933,217 +933,217 @@
         "displayName": "Full circle",
         "iconName": "full-circle",
-        "tags": "dot, point",
+        "tags": "dot, point, orb, black",
```
```suggestion
        "tags": "dot, point, orb, filled",
```

icons may have any color, and they are white(-ish) in dark theme

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1157,63 +1157,63 @@
         "displayName": "Comment",
         "iconName": "comment",
-        "tags": "statement, discussion, opinion, view",
+        "tags": "statement, discussion, opinion, view, chat",
```
```suggestion
        "tags": "statement, discussion, opinion, view, chat, message, typing",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1157,63 +1157,63 @@
         "displayName": "Circle arrow: right",
         "iconName": "circle-arrow-right",
-        "tags": "direction, east",
+        "tags": "direction, east, go, monocle, next, submit, right arrow",
```
```suggestion
        "tags": "direction, east, go, next, submit, right arrow",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1157,63 +1157,63 @@
         "displayName": "Circle arrow: left",
         "iconName": "circle-arrow-left",
-        "tags": "direction, west",
+        "tags": "direction, west, return, backspace, back, left arrow circle, arrow, left, go left",
```
```suggestion
        "tags": "direction, west, return, backspace, back, go left",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1157,63 +1157,63 @@
         "displayName": "Circle arrow: up",
         "iconName": "circle-arrow-up",
-        "tags": "direction, north",
+        "tags": "direction, north, upload, upward, above, push, promote, publish, up, arrow, sort",
```
```suggestion
        "tags": "direction, north, upload, upward, above, push, promote, publish",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1157,63 +1157,63 @@
         "displayName": "Circle arrow: down",
         "iconName": "circle-arrow-down",
-        "tags": "direction, south",
+        "tags": "direction, south, down arrow, arrow down, below, download, save to device, keep offline, down, next (if the next step is below)",
```
```suggestion
        "tags": "direction, south, below, download, save to device, keep offline, next",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Cloud: download",
         "iconName": "cloud-download",
-        "tags": "file, storage, transfer",
+        "tags": "file, storage, transfer, download, cloud",
```
```suggestion
        "tags": "file, storage, transfer",
```

redundant with icon name

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Cloud: upload",
         "iconName": "cloud-upload",
-        "tags": "file, storage, transfer",
+        "tags": "file, storage, transfer, export, upload, upload to cloud",
```
```suggestion
        "tags": "file, storage, transfer, export",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Move",
         "iconName": "move",
-        "tags": "arrows, directions, position, location",
+        "tags": "arrows, directions, position, location, fractal, drag",
```
```suggestion
        "tags": "arrows, directions, position, location, drag, cursor",
```

not sure about "fractal" in this context

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Chevron: left",
         "iconName": "chevron-left",
-        "tags": "arrow, direction",
+        "tags": "arrow, direction, left chevron, open angle bracket, open tag",
```
```suggestion
        "tags": "arrow, direction, open angle bracket, open tag",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Chevron: right",
         "iconName": "chevron-right",
-        "tags": "arrow, direction",
+        "tags": "arrow, direction, next, right arrow, forward",
```
```suggestion
        "tags": "arrow, direction, next, forward, close angle bracket, close tag",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Chevron: up",
         "iconName": "chevron-up",
-        "tags": "arrow, direction",
+        "tags": "arrow, direction, up, collapse, see here, exponent, icons are expensive so look at this key from my keyboard",
```
```suggestion
        "tags": "arrow, direction, collapse, see here, exponent",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Chevron: down",
         "iconName": "chevron-down",
-        "tags": "arrow, direction",
+        "tags": "arrow, direction, carrot down, arrow down, below, palantir, down, chevron",
```
```suggestion
        "tags": "arrow, direction, below",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Random",
         "iconName": "random",
-        "tags": "arrows, aim",
+        "tags": "arrows, aim, direction, arrow, change",
```
```suggestion
        "tags": "arrows, aim, direction, change, shuffle, mix, swap",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1227,91 +1227,91 @@
         "displayName": "Login",
         "iconName": "log-in",
-        "tags": "arrow, sign in",
+        "tags": "arrow, sign in, save, exit, logout",
```
```suggestion
        "tags": "arrow, sign in, auth, authentication",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1332,42 +1332,42 @@
         "displayName": "Ban circle",
         "iconName": "ban-circle",
-        "tags": "circle, refusal",
+        "tags": "circle, refusal, do not enter, inaccessible, screw head, pause, slow, null, subtract, minus, delete",
```
```suggestion
        "tags": "circle, refusal, do not enter, inaccessible, screw head, flathead, slow, null, subtract, minus, delete",
```

not sure about "pause" in this context

gabrieljroth replied:
agreed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1332,42 +1332,42 @@
         "displayName": "Mobile video",
         "iconName": "mobile-video",
-        "tags": "film, broadcast, television",
+        "tags": "film, broadcast, television, video, recording, video call, video camera, triangle enter rectangle",
```
```suggestion
        "tags": "film, broadcast, television, video, recording, video call, camera, camcorder",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1381,21 +1381,21 @@
         "displayName": "Volume: off",
         "iconName": "volume-off",
-        "tags": "audio, video, speaker, music, sound, low",
+        "tags": "audio, video, speaker, music, sound, low, volume",
```
```suggestion
        "tags": "audio, video, speaker, music, sound, low, mute, silence",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1381,21 +1381,21 @@
         "displayName": "Volume: down",
         "iconName": "volume-down",
-        "tags": "audio, video, speaker, music, sound",
+        "tags": "audio, video, speaker, music, sound, volume, low voume, quiet, small-sound",
```
```suggestion
        "tags": "audio, video, speaker, music, sound, quiet, lower",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1381,21 +1381,21 @@
         "displayName": "Volume: up",
         "iconName": "volume-up",
-        "tags": "audio, video, speaker, music, sound, high",
+        "tags": "audio, video, speaker, music, sound, high, sound on",
```
```suggestion
        "tags": "audio, video, speaker, music, sound, high, on, increase, loud",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1430,49 +1430,49 @@
         "displayName": "Stop",
         "iconName": "stop",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, box, square, error, not possible, blocked, black square, shape",
```
```suggestion
        "tags": "player, media, controls, digital, analogue, film, audio, video, box, square, error, not possible, blocked, shape, filled",
```

see my other comment about shape color "black"

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1430,49 +1430,49 @@
         "displayName": "Eject",
         "iconName": "eject",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, triangle, rectangle, up triangle, up, arrow, upload",
```
```suggestion
        "tags": "player, media, controls, digital, analogue, film, audio, video",
```

I don't think any of these actually help, there other better triangle/arrow icons

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Glass",
         "iconName": "glass",
-        "tags": "glassware, drink",
+        "tags": "glassware, drink, cheers, cup, wine glass, wine, chalice, goblet",
```
```suggestion
        "tags": "glassware, drink, cheers, cup, wine, chalice, goblet",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Flash",
         "iconName": "flash",
-        "tags": "light, contrast, photograph, picture",
+        "tags": "light, contrast, photograph, picture, sun, star, illuminate, examine, brighten",
```
```suggestion
        "tags": "light, contrast, photograph, picture, sun, star, illuminate, examine, brighten, brightness, exposure",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Font",
         "iconName": "font",
-        "tags": "scale, typography, size",
+        "tags": "scale, typography, size, a, letter, alphabet, letter a, area",
```
```suggestion
        "tags": "scale, typography, size, letter, alphabet",
```

"a" is not going to help anyone find an icon via search

gabrieljroth replied:
agreed!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Saved",
         "iconName": "saved",
-        "tags": "document, check mark, tick",
+        "tags": "document, check mark, tick, approval, document ready, checks complete, checklist, approved document, completed document, verified document, confirm, valid, save, upload finished, confirmation, validation, checked, approved, done document, completed, certificate",
```
```suggestion
        "tags": "document, check mark, tick, approval, checks complete, checklist, approved, verified, confirm, valid, save, finished, confirmation, validation, checked, approved, done, ready, completed, certificate",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Book",
         "iconName": "book",
-        "tags": "pages, album, brochure, manual",
+        "tags": "pages, album, brochure, manual, bookmark, notebook, journal, documentation, reading, instructions, file, cathedral",
```
```suggestion
        "tags": "pages, album, brochure, manual, bookmark, notebook, journal, documentation, reading, instructions, file",
```

I'm not sure how cathedral relates...?

gabrieljroth replied:
i guess it kind of looks like one lol

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Hand: right",
         "iconName": "hand-right",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, hand, point right, finger, tip, attention",
```
```suggestion
        "tags": "gesture, direction, point, pointer, pointing, finger, tip, attention",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Hand: up",
         "iconName": "hand-up",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, pointer, forefinger, right hand, click, point, cursor",
```
```suggestion
        "tags": "gesture, direction, point, pointer, pointing, finger, forefinger, click, cursor",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Hand: down",
         "iconName": "hand-down",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, point down, hand, pointer finger, hand pointing down, select, down",
```
```suggestion
        "tags": "gesture, direction, point, pointer, pointing, finger, select",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Hand: left",
         "iconName": "hand-left",
-        "tags": "gesture, direction",
+        "tags": "gesture, direction, point, finger, this, here, click",
```
```suggestion
        "tags": "gesture, direction, point, pointer, pointing, finger, this, here, click",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Thumbs: down",
         "iconName": "thumbs-down",
-        "tags": "hand, dislike, bad",
+        "tags": "hand, dislike, bad, thumb down",
```
```suggestion
        "tags": "hand, dislike, bad",
```

this addition doesn't change the search results

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1486,126 +1486,126 @@
         "displayName": "Thumbs: up",
         "iconName": "thumbs-up",
-        "tags": "hand, like, ok",
+        "tags": "hand, like, ok, good, okay",
```
```suggestion
        "tags": "hand, like, ok, good, okay",
```

adidahiya reviewed (changes requested):
reviewed up to line 1600

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Apply suggestions from code review

Great edits! Thanks so much :)

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Apply suggestions from code review

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

Commit:
edited eye-open tag

Commit:
Merge branch 'groth/add-tags' of https://github.com/gabrieljroth/blueprint into groth/add-tags

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1591,21 +1591,21 @@
         "displayName": "Shopping cart",
         "iconName": "shopping-cart",
-        "tags": "trolley, mall, online, store, business",
+        "tags": "trolley, mall, online, store, business, cart, shopping",
```
```suggestion
        "tags": "trolley, mall, online, store, business",
```

redundant tags

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Undo",
         "iconName": "undo",
-        "tags": "back, cancel, reverse, revoke,",
+        "tags": "back, cancel, reverse, revoke, , rollback, revert",
```
```suggestion
        "tags": "back, cancel, reverse, revoke, rollback, revert",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Layout: linear",
         "iconName": "layout-linear",
-        "tags": "dots, connection, line",
+        "tags": "dots, connection, line, pipeline, three dots in a line, line of circles, beads, three dots, dot dot dot, more, daisy, connect, graph",
```
```suggestion
        "tags": "dot, three, dots, connection, line, pipeline, line of circles, beads, more, daisy, connect, graph",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Double caret: vertical",
         "iconName": "double-caret-vertical",
-        "tags": "sort, arrow, list",
+        "tags": "sort, arrow, list, up and down arrows, vertical arrows, elevator, scroll, navigate, expand, up, down, arrows",
```
```suggestion
        "tags": "sort, arrow, list, elevator, scroll, navigate, expand, up, down, vertical, arrows",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Double caret: horizontal",
         "iconName": "double-caret-horizontal",
-        "tags": "sort, arrow, list",
+        "tags": "sort, arrow, list, music, play, navigate, expand, slider, left right, arrows",
```
```suggestion
        "tags": "sort, arrow, list, navigate, expand, slider, left, right, arrows",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Sort: numerical descending",
         "iconName": "sort-numerical-desc",
-        "tags": "order, list, array, arrange",
+        "tags": "order, list, array, arrange, descending, descending order, sort descending, sort, descending sort",
```
```suggestion
        "tags": "order, list, array, arrange, descending",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Contrast",
         "iconName": "contrast",
-        "tags": "color, brightness",
+        "tags": "color, brightness, settings, light mode/dark mode, cog, light/dark mode",
```
```suggestion
        "tags": "color, brightness, settings, light, dark, mode, theme, cog",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Area chart",
         "iconName": "timeline-area-chart",
-        "tags": "graph, line, diagram",
+        "tags": "graph, line, diagram, trend up, cubic, chart, area chart, growth, increasing graph",
```
```suggestion
        "tags": "graph, line, diagram, trend up, growth, increasing",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Doughnut chart",
         "iconName": "doughnut-chart",
-        "tags": "circle, section, part, graph",
+        "tags": "circle, section, part, graph, quarter, pie chart, storage space, usage, statistics",
```
```suggestion
        "tags": "circle, section, part, graph, quarter, pie chart, storage space, usage, statistics, donut, visualization",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1619,119 +1619,119 @@
         "displayName": "Add to folder",
         "iconName": "add-to-folder",
-        "tags": "file, portfolio, case, import",
+        "tags": "file, portfolio, case, import, folder, open folder, open",
```
```suggestion
        "tags": "file, portfolio, case, import, open",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Sort: ascending",
         "iconName": "sort-asc",
-        "tags": "order, list, array, arrange",
+        "tags": "order, list, array, arrange, ascending order, increasing order, histogram, ascending, queue",
```
```suggestion
        "tags": "order, list, array, arrange, ascending, increasing, histogram, queue",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Sort: descending",
         "iconName": "sort-desc",
-        "tags": "order, list, array, arrange",
+        "tags": "order, list, array, arrange, sort, decreasing",
```
```suggestion
        "tags": "order, list, array, arrange, descending, decreasing",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Small cross",
         "iconName": "small-cross",
-        "tags": "cross mark, fail, delete, no, close, remove",
+        "tags": "cross mark, fail, delete, no, close, remove, x, cross, cancel, exit",
```
```suggestion
        "tags": "cross mark, fail, delete, no, close, remove, x, cancel, exit",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Small tick",
         "iconName": "small-tick",
-        "tags": "mark, sign, ok, approved, success",
+        "tags": "mark, sign, ok, approved, success, check, complete, agree, verify, yes, correct, tick",
```
```suggestion
        "tags": "mark, sign, ok, approved, success, check, complete, agree, verify, yes, correct",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Power",
         "iconName": "power",
-        "tags": "button, on, off",
+        "tags": "button, on, off, power on, power off",
```
```suggestion
        "tags": "button, on, off, toggle",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Column layout",
         "iconName": "column-layout",
-        "tags": "layout, arrangement",
+        "tags": "layout, arrangement, documents, sidebar, side menu, side bar, panel, gui, table, section, columns, tables",
```
```suggestion
        "tags": "layout, arrangement, sidebar, side menu, side bar, panel, gui, table, section, columns, tables",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Arrow: top left",
         "iconName": "arrow-top-left",
-        "tags": "direction, north west",
+        "tags": "direction, north west, arrow, up left arrow, top left arrow, cursor, up left, arrow up left",
```
```suggestion
        "tags": "direction, north west, cursor, up left",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Arrow: top right",
         "iconName": "arrow-top-right",
-        "tags": "direction, north east",
+        "tags": "direction, north east, top right arrow, upper right arrow, arrow, top right, pointer, up, go, arrow diagonal, open in new tab, arrow up right, upper right corner",
```
```suggestion
        "tags": "direction, north east, top right, pointer, up, diagonal, open in new tab, corner",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Arrow: bottom right",
         "iconName": "arrow-bottom-right",
-        "tags": "direction, south east",
+        "tags": "direction, south east, arrow down right, down left arrow, diagonal arrow, diagonal down left arrow",
```
```suggestion
        "tags": "direction, south east, diagonal, down right",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Arrow: bottom left",
         "iconName": "arrow-bottom-left",
-        "tags": "direction, south west",
+        "tags": "direction, south west, arrow, bottom left direction, point, minimize, collapse, go to, here, this, down and to the left, right down arrow, diagonal",
```
```suggestion
        "tags": "direction, south west, arrow, diagonal, down left",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Mugshot",
         "iconName": "mugshot",
-        "tags": "person, photograph, picture,",
+        "tags": "person, photograph, picture, portrait, profile picture, mark zuckerberg, user",
```
```suggestion
        "tags": "person, photograph, picture, portrait, profile, user",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Text highlight",
         "iconName": "text-highlight",
-        "tags": "selector, content",
+        "tags": "selector, content, type, input, text insert, select, highlight, text input, text box, input text, search, enter text, text area",
```
```suggestion
        "tags": "selector, content, type, input, insert, select, box, search, text area, textarea",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Chevron: forward",
         "iconName": "chevron-forward",
-        "tags": "skip, direction",
+        "tags": "skip, direction, skip to end, strikeout",
```
```suggestion
        "tags": "skip, direction, end",
```

what does "strikeout" mean in this context?

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Rotate: page",
         "iconName": "rotate-page",
-        "tags": "turn, anti clockwise",
+        "tags": "turn, anti clockwise, rotate, rotate counterclockwise, rotate left",
```
```suggestion
        "tags": "turn, anti clockwise, counterclockwise, left",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Badge",
         "iconName": "badge",
-        "tags": "emblem, symbol, identification, insignia, marker",
+        "tags": "emblem, symbol, identification, insignia, marker, protect, privacy, shield, law enforcement, police, sheriff, rule enforcement, shield or protection, security",
```
```suggestion
        "tags": "emblem, symbol, identification, insignia, marker, protect, privacy, shield, law enforcement, police, sheriff, rule enforcement, security",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Function",
         "iconName": "function",
-        "tags": "math, calculation",
+        "tags": "math, calculation, logic, fx",
```
```suggestion
        "tags": "math, calculation, logic, fx, formula, spreadsheet",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Waterfall chart",
         "iconName": "waterfall-chart",
-        "tags": "graph, diagram",
+        "tags": "graph, diagram, waterfall, chart, bar chart, yield",
```
```suggestion
        "tags": "graph, diagram, bar chart, yield",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Stacked chart",
         "iconName": "stacked-chart",
-        "tags": "bar chart",
+        "tags": "bar chart, stacked bar graph, graph, sort, filter, chart, statistics, summary",
```
```suggestion
        "tags": "bar chart, graph, statistics, summary",
```

by this logic... we would add "statistics" to every chart icon... just something to think about re: consistency

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "New person",
         "iconName": "new-person",
-        "tags": "person, human, male, female, character, customer, individual, add",
+        "tags": "person, human, male, female, character, customer, individual, add, add user, new user, additional user, add person",
```
```suggestion
        "tags": "person, human, male, female, character, customer, individual, add, user, additional",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Pivot table",
         "iconName": "pivot-table",
-        "tags": "rotate, axis",
+        "tags": "rotate, axis, hinge, corner, pivot, lift, open, lever",
```
```suggestion
        "tags": "rotate, axis, hinge, corner, lift, open, lever",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Segmented control",
         "iconName": "segmented-control",
-        "tags": "button, switch, option",
+        "tags": "button, switch, option, switch button toggle, toggle, progress bar",
```
```suggestion
        "tags": "button, switch, option, toggle, progress bar",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,217 +1745,217 @@
         "displayName": "Filter: list",
         "iconName": "filter-list",
-        "tags": "filtering, funnel, tube, pipe",
+        "tags": "filtering, funnel, tube, pipe, filter",
```
```suggestion
        "tags": "filtering, funnel, tube, pipe",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Annotation",
         "iconName": "annotation",
-        "tags": "note, comment, edit,",
+        "tags": "note, comment, edit, , edit, write",
```
```suggestion
        "tags": "note, comment, edit, write",
```

adidahiya replied:
@gabrieljroth could you please save me a bit of review time and do another pass on these edits to make sure you don't have empty tags or redundant words like this in the rest of this PR?

gabrieljroth replied:
Yes of course. Sorry about that!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Ring",
         "iconName": "ring",
-        "tags": "empty, circle, selection",
+        "tags": "empty, circle, selection, hollow circle, empty circle, hole, disc",
```
```suggestion
        "tags": "empty, circle, selection, hollow, empty, hole, disc, donut",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Heat grid",
         "iconName": "heat-grid",
-        "tags": "chart",
+        "tags": "chart, grid, heat map, squares, black and white",
```
```suggestion
        "tags": "chart, map, squares, black and white",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Variable",
         "iconName": "variable",
-        "tags": "math, calculation",
+        "tags": "math, calculation, function input, variable input, parentheses, function, parameter, input",
```
```suggestion
        "tags": "math, calculation, function, input, parentheses, parameter",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Add row: top",
         "iconName": "add-row-top",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add, add row",
```
```suggestion
        "tags": "table, attach, join",
```

here's another example -- now that we've gone through a few rounds of review, I'd appreciate if you could do some self-review of the remaining edits and make sure we don't have redundant tags like this. thanks 🙏🏽

gabrieljroth replied:
Will do!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Add row: bottom",
         "iconName": "add-row-bottom",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, widget, add",
```
```suggestion
        "tags": "table, attach, join, widget",
```

if this is a "widget", shouldn't "add-row-top" also be a widget? same for add-*-column icons. let's be consistent.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Add column: left",
         "iconName": "add-column-left",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add tab, add tab left, add window, add column, add right",
```
```suggestion
        "tags": "table, attach, join, tab, window, section",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Add column: right",
         "iconName": "add-column-right",
-        "tags": "table, attach, join",
+        "tags": "table, attach, join, add section right, add, new, medicine",
```
```suggestion
        "tags": "table, attach, join, tab, window, section",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Remove row: bottom",
         "iconName": "remove-row-bottom",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, box, files, dashboard, application, container, oven",
```
```suggestion
        "tags": "table, detach, delete, box, files, container, oven",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Remove column: left",
         "iconName": "remove-column-left",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, door, open, box, application, sidebar, dashboard",
```
```suggestion
        "tags": "table, detach, delete, sidebar, tab, window, section",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Remove column: right",
         "iconName": "remove-column-right",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, delete, module, section, widget, remove panel, remove widget, divide panel, library, book, collection, drawer, sidebar, menu",
```
```suggestion
        "tags": "table, detach, delete, sidebar, section, widget, panel",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Double chevron: left",
         "iconName": "double-chevron-left",
-        "tags": "arrows, multiple, direction",
+        "tags": "arrows, multiple, direction, chevrons left, expand left, shift left, double left chevron, go left, inbox, new email, skip backward, quick reverse, move left, left arrows, open bracket, back",
```
```suggestion
        "tags": "arrows, multiple, direction, expand, shift, go left, skip backward, quick reverse, move, open bracket, back",
```

not sure what this has to do with emails

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Double chevron: right",
         "iconName": "double-chevron-right",
-        "tags": "arrows, multiple, direction",
+        "tags": "arrows, multiple, direction, next, fast forward, two right arrows, advance, go to end, forward, continue, right, go, double arrow right",
```
```suggestion
        "tags": "arrows, multiple, direction, next, fast forward, advance, continue, go, end, double arrow",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Double chevron: up",
         "iconName": "double-chevron-up",
-        "tags": "arrows, multiple, direction",
+        "tags": "arrows, multiple, direction, skip up, increase a lot, forward, fast forward, power up, up, upload, swipe up, double up, roofs, collapse upwards",
```
```suggestion
        "tags": "arrows, multiple, direction, skip up, increase, power up, upload, swipe up, double, roof, collapse, upwards",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1969,182 +1969,182 @@
         "displayName": "Double chevron: down",
         "iconName": "double-chevron-down",
-        "tags": "arrows, multiple, direction",
+        "tags": "arrows, multiple, direction, down, downward, downgrade",
```
```suggestion
        "tags": "arrows, multiple, direction, downward, downgrade",
```

adidahiya reviewed (changes requested):
reviewed up to line 2100

Commit:
edited direction-left and direction-right tags

gabrieljroth committed:
Apply suggestions from code review

First pass at new commit suggestions. Next will review the rest and clean things up on my end.

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth committed:
Apply suggestions from code review

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

Commit:
proofread lines 2100 through to the end, removing redundant tags, empty tags, mispelled tags, etc.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2116,35 +2116,35 @@
         "displayName": "Key: shift",
         "iconName": "key-shift",
-        "tags": "interface, shortcuts, buttons",
+        "tags": "interface, shortcuts, buttons, look up, look at this, this is my tiny house",
```
```suggestion
        "tags": "interface, shortcuts, buttons, look up, look at this",
```

gabrieljroth replied:
sorry i missed this lol

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Drag handle: horizontal",
         "iconName": "drag-handle-horizontal",
-        "tags": "move, pull",
+        "tags": "move, pull, menu, more, options, set, multiple, handle, grab, dots",
```
```suggestion
        "tags": "move, pull, set, multiple, handle, grab, dots",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Curved range chart",
         "iconName": "curved-range-chart",
-        "tags": "graph, diagram",
+        "tags": "graph, diagram, analsysis, model, regression",
```
```suggestion
        "tags": "graph, diagram, analysis, model, regression",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Horizontal bar chart: descending",
         "iconName": "horizontal-bar-chart-desc",
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, increase, menu, descending, sort, decreasing, list, left align, left indent, left align",
```
```suggestion
        "tags": "graph, histogram, increase, descending, sort, decreasing, list",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Document: open",
         "iconName": "document-open",
-        "tags": "paper, access",
+        "tags": "paper, access, arrow, file, export, download",
```
looks like more of an "import" to me, especially if document-share is getting tagged with "export"

<img width="298" alt="image" src="https://user-images.githubusercontent.com/723999/189708424-f8cad7c3-7dfb-4c24-9466-54002f69835f.png">

```suggestion
        "tags": "paper, access, arrow, file, import",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: left",
         "iconName": "alignment-left",
-        "tags": "layout, position",
+        "tags": "layout, position, bar chart, histogram",
```
```suggestion
        "tags": "layout, position",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: vertical center",
         "iconName": "alignment-vertical-center",
-        "tags": "layout, position",
+        "tags": "layout, position, volume, bars, biplane",
```
```suggestion
        "tags": "layout, position, bars, biplane",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: right",
         "iconName": "alignment-right",
-        "tags": "layout, position",
+        "tags": "layout, position, graph, chart",
```
```suggestion
        "tags": "layout, position",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: top",
         "iconName": "alignment-top",
-        "tags": "layout, position",
+        "tags": "layout, position, graph, chart",
```
```suggestion
        "tags": "layout, position",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: horizontal center",
         "iconName": "alignment-horizontal-center",
-        "tags": "layout, position",
+        "tags": "layout, position, drone, airplane, satellite, plug, statistics, shadow analysis, time-series",
```
```suggestion
        "tags": "layout, position, drone, airplane, satellite, statistics, shadow analysis",
```

am I missing something with "plug" and "time series"? it doesn't look like either of those to me.

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Alignment: bottom",
         "iconName": "alignment-bottom",
-        "tags": "layout, position",
+        "tags": "layout, position, trend, comparison, graph, bar chart, analytics, histogram, bars",
```
as I've just suggested on the other "alignment" icon edits, I don't see a need to make these icons be dual-purpose and overlap in semantics with the chart icons. we already have plenty of chart icons.

```suggestion
        "tags": "layout, position, bars",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Git: merge",
         "iconName": "git-merge",
-        "tags": "github, repository, code, command",
+        "tags": "github, repository, code, command, branch, git, merge, version",
```
```suggestion
        "tags": "github, repository, code, command, branch, version control",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Git: pull",
         "iconName": "git-pull",
-        "tags": "github, repository, code, command",
+        "tags": "github, repository, code, command, pull request, insert, version control, merge",
```
```suggestion
        "tags": "github, repository, code, command, pull request, branch, insert, version control, merge",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Git: branch",
         "iconName": "git-branch",
-        "tags": "github, repository, code, command",
+        "tags": "github, repository, code, command, branch, git, version, fork",
```
```suggestion
        "tags": "github, repository, code, command, version control, fork",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Build",
         "iconName": "build",
-        "tags": "hammer, tool",
+        "tags": "hammer, tool, force build",
```
```suggestion
        "tags": "hammer, tool, force, work",
```

gabrieljroth replied:
what's wrong with "build"? I like both work and build

adidahiya replied:
"build" is already the name of the icon :)

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Symbol: circle",
         "iconName": "symbol-circle",
-        "tags": "shape, figure",
+        "tags": "shape, figure, dot, black, period, spot",
```
```suggestion
        "tags": "shape, figure, dot, filled, period, spot",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Symbol: square",
         "iconName": "symbol-square",
-        "tags": "shape, figure",
+        "tags": "shape, figure, record, stop, dot",
```
```suggestion
        "tags": "shape, figure, stop",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Oil field",
         "iconName": "oil-field",
-        "tags": "fuel, petroleum, gas, well, drilling, pump",
+        "tags": "fuel, petroleum, gas, well, drilling, pump, wheel, hammer, engineering, refinery, well",
```
duplicate

```suggestion
        "tags": "fuel, petroleum, gas, well, drilling, pump, wheel, hammer, engineering, refinery",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "New map",
         "iconName": "map-create",
-        "tags": "map, location, position, geography, world",
+        "tags": "map, location, position, geography, world, new brochure, new pamphlet, navigation, travel, gps, coalesce",
```
"coalesce" feels like a stretch here

```suggestion
        "tags": "map, location, position, geography, world, new brochure, new pamphlet, navigation, travel, gps",
```

gabrieljroth replied:
agreed

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "New grid item",
         "iconName": "new-grid-item",
-        "tags": "layout, arrangement, add",
+        "tags": "layout, arrangement, add, squares, point, target, microsoft, no idea, block",
```
```suggestion
        "tags": "layout, arrangement, add, squares, target, microsoft, block, quadrant",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2193,406 +2193,406 @@
         "displayName": "Git: repo",
         "iconName": "git-repo",
-        "tags": "github, repository, code, command",
+        "tags": "github, repository, code, command, desktop, screen, entry, no idea, bookmark, saved page, notepad, book, dictionary, notebook, file, dossier, section",
```
```suggestion
        "tags": "github, repository, code, command, bookmark, saved page, notepad, book, dictionary, notebook, file, dossier",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2606,161 +2606,161 @@
         "displayName": "Horizontal bar chart: ascending",
         "iconName": "horizontal-bar-chart-asc",
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, sort, ranking, stats, review",
```
not sure what this has to do with "review"

```suggestion
        "tags": "graph, histogram, sort, ranking, stats",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2606,161 +2606,161 @@
         "displayName": "Blocked person",
         "iconName": "blocked-person",
-        "tags": "person, human, male, female, character, customer, individual, social, banned, prohibited",
+        "tags": "human, male, female, character, customer, individual, social, banned, prohibited, box remove, box minus",
```
what does "box" mean here? doesn't seem relevant

```suggestion
        "tags": "human, male, female, character, customer, individual, social, banned, prohibited",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2606,161 +2606,161 @@
         "displayName": "Satellite",
         "iconName": "satellite",
-        "tags": "communication, space",
+        "tags": "communication, space, radar, sensor, airplane, wifi, landing",
```
```suggestion
        "tags": "communication, space, radar, sensor, airplane, data, gps, orbit, landing",
```

gabrieljroth replied:
I like wifi (unless we have a separate wifi icon which i dont think we do)
This would be a good icon option if i wanted to represent wifi
I do like your other additions though!

adidahiya replied:
wifi almost always comes from land-based signals, so I consider that distinct from satellite-enabled communication.

it is unfortunate that we don't have a good "wifi" icon. I still don't think "satellite" is the best one for that, maybe "antenna" is better. but I suppose in the meantime while we don't have a good "wifi" icon we can leave this tag here.

```suggestion
        "tags": "communication, space, radar, sensor, airplane, wifi, data, gps, orbit, landing",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2774,28 +2774,28 @@
         "displayName": "Cell tower",
         "iconName": "cell-tower",
-        "tags": "signal, communication, radio, mast",
+        "tags": "signal, communication, radio, mast, connection, aerial, radar, connectivity, cancer, internet",
```
how does "cancer" relate to this icon?

```suggestion
        "tags": "signal, communication, radio, mast, connection, aerial, radar, connectivity, internet, data, transmission, broadcast",
```

gabrieljroth replied:
I think they were thinking radiation --> cancer. we can remove

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2774,28 +2774,28 @@
         "displayName": "IP address",
         "iconName": "ip-address",
-        "tags": "internet, protocol, number, id, network",
+        "tags": "internet, protocol, number, network, geolocation, local intelectual property, isp, internet service provider, interp",
```
```suggestion
        "tags": "internet, protocol, number, network, geolocation, local, intellectual property, isp, internet service provider",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2809,70 +2809,70 @@
         "displayName": "Panel: stats",
         "iconName": "panel-stats",
-        "tags": "sidebar, layout, list",
+        "tags": "sidebar, layout, list, window, side bar, menu, website, dashboard, tablet, report, dataset preview",
```
```suggestion
        "tags": "sidebar, layout, list, window, side bar, drawer, menu, website, dashboard, tablet, report, dataset preview",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2809,70 +2809,70 @@
         "displayName": "Prescription",
         "iconName": "prescription",
-        "tags": "instruction, direction, medicine, drug, medication, mixture",
+        "tags": "instruction, direction, medicine, drug, medication, mixture, reaction",
```
```suggestion
        "tags": "instruction, direction, medicine, drug, medication, mixture, reaction, pharmacy",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2886,168 +2886,168 @@
         "displayName": "Widget: button",
         "iconName": "widget-button",
-        "tags": "element, click, press",
+        "tags": "element, click, press, loading, more, details, options, breadcrumbs, additional, lego, dots, edit",
```
```suggestion
        "tags": "element, click, press, loading, more, details, options, breadcrumbs, additional, lego, dots",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2886,168 +2886,168 @@
         "displayName": "Widget: header",
         "iconName": "widget-header",
-        "tags": "element, layout, top",
+        "tags": "element, layout, top, refrigerator, section, div, container, split horizontal, tab, column, note",
```
```suggestion
        "tags": "element, layout, top, refrigerator, fridge, section, div, container, split horizontal, tab, column",
```

gabrieljroth replied:
i like the addition :)

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2886,168 +2886,168 @@
         "displayName": "Widget: footer",
         "iconName": "widget-footer",
-        "tags": "element, layout, bottom",
+        "tags": "element, layout, bottom, window, book, guillotine",
```
```suggestion
        "tags": "element, layout, bottom, window, section, div, container, guillotine",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2886,168 +2886,168 @@
         "displayName": "Series: search",
         "iconName": "series-search",
-        "tags": "timeseries, plot, line, data, chart, find, filter",
+        "tags": "timeseries, plot, line, data, chart, graph, find, investiage, inspect, finder, looker, trend, analyze, expand",
```
```suggestion
        "tags": "timeseries, plot, line, data, chart, graph, find, investigate, inspect, finder, looker, trend, analyze",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -2886,168 +2886,168 @@
         "displayName": "Join: inner",
         "iconName": "inner-join",
-        "tags": "circles, combine, connect, add, part, slice",
+        "tags": "circles, combine, connect, add, part, slice, eye, cell, intersection, overlap, option, navigate, vision, venn diagram",
```
we already have a good "eye" icon so I don't think we need to conflate this with that one.

```suggestion
        "tags": "circles, combine, connect, add, part, slice, cell, intersection, overlap, venn diagram",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Tree diagram",
         "iconName": "diagram-tree",
-        "tags": "organization, chart, hierarchy, relationship, reference",
+        "tags": "organization, chart, hierarchy, relationship, reference, eye, heirarchy, org chart",
```
```suggestion
        "tags": "organization, chart, org, hierarchy, relationship, reference",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Flow: review branch",
         "iconName": "flow-review-branch",
-        "tags": "workflow, edge, node, fork",
+        "tags": "workflow, edge, node, fork, locate, axis, graph, outward, away, location, relationship, links, unknown, geolocate, map, grappling hook",
```
there are a decent number of "locate" / "location" icons already, so I don't think we need to conflate these with map/location terminology

```suggestion
        "tags": "workflow, edge, node, fork, axis, graph, outward, away, location, relationship, links, unknown, grappling hook",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Table: filtered",
         "iconName": "th-filtered",
-        "tags": "index, rows, columns, agenda, list, spreadsheet, filtering, funnel",
+        "tags": "index, rows, columns, agenda, list, spreadsheet, funnel, spreadsheet, excel, sort, excel",
```
```suggestion
        "tags": "index, rows, columns, agenda, list, spreadsheet, funnel, spreadsheet, excel, sort",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Lifesaver",
         "iconName": "lifesaver",
-        "tags": "help, support, aid, advice",
+        "tags": "help, support, aid, advice, poker, casino, circle",
```
```suggestion
        "tags": "help, support, aid, advice, poker, casino, chip, circle",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Inbox: geo",
         "iconName": "inbox-geo",
-        "tags": "folder, mail, file, message, geography, location, area, globe",
+        "tags": "folder, mail, file, message, geography, location, area, globe, save map, save location, add location, international mail, upload",
```
```suggestion
        "tags": "folder, mail, file, message, geography, location, area, globe, save map, save location, add location, international",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Signal search",
         "iconName": "signal-search",
-        "tags": "signal, communication, radio, mast",
+        "tags": "communication, radio, mast, towers, find telecom,",
```
```suggestion
        "tags": "communication, radio, mast, towers, find, telecom, broadcast",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Hat",
         "iconName": "hat",
-        "tags": "head, clothing, cap, fedora, role, level, permission",
+        "tags": "head, clothing, cap, fedora, role, level, permission, detective, inspector gadget",
```
```suggestion
        "tags": "head, clothing, cap, fedora, role, level, permission, detective, inspector",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Cycle",
         "iconName": "cycle",
-        "tags": "cycle, bike, cyclist, biker, ride, rider, transportation",
+        "tags": "bike, cyclist, biker, rider, transportation, rossing, pedestrian, bicyclist, cyling, motorcyle",
```
```suggestion
        "tags": "bike, cyclist, biker, rider, transportation, crossing, pedestrian, bicyclist, cycling, motorcycle",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Modal",
         "iconName": "modal",
-        "tags": "modal, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, browser, window, app",
```
```suggestion
        "tags": "application, layout, user-interface, UI, browser, window, app, dialog",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Modal filled",
         "iconName": "modal-filled",
-        "tags": "modal, application, layout, user-interface, UI",
+        "tags": "modal, application, layout, user-interface, UI, window, computer, app",
```
```suggestion
        "tags": "application, layout, user-interface, UI, window, app, dialog",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Drawer left",
         "iconName": "drawer-left",
-        "tags": "drawer, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, pass, export, transform, move, sliding door, flow right block",
```
what does "pass" mean in this context?

gabrieljroth replied:
i guess like passing an object to someone? You can remove

adidahiya replied:
yeah let's remove `pass`

```suggestion
        "tags": "application, layout, user-interface, UI, export, transform, move, sliding door, flow right block",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Drawer left filled",
         "iconName": "drawer-left-filled",
-        "tags": "drawer, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, right, shift right, close door, right justify, drag right, output, logout, rectangle arrow",
```
```suggestion
        "tags": "application, layout, user-interface, UI, right, shift, close door, justify, drag, move, output, rectangle arrow",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Drawer right",
         "iconName": "drawer-right",
-        "tags": "drawer, application, layout, user-interface, UI",
+        "tags": "application, layout, user-interface, UI, left, door, arrow, move, shift, back, close, revert, return, remove, collapse",
```
```suggestion
        "tags": "application, layout, user-interface, UI, left, door, arrow, move, shift, back, close, collapse",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Send message",
         "iconName": "send-message",
-        "tags": "send, deliver, paper-airplane, post",
+        "tags": "deliver, paper-airplane, post, next, arrow, continue",
```
```suggestion
        "tags": "deliver, paper-airplane, post, mail",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Data Connection",
         "iconName": "data-connection",
-        "tags": "connectivity, connection, database, data, status, health",
+        "tags": "connectivity, database, status, health, blank, nothing, empty",
```
```suggestion
        "tags": "connectivity, database, status, health, blank",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Switch",
         "iconName": "switch",
-        "tags": "switch, electric, electrical",
+        "tags": "electrical, broken graph, off, disconnect, open, broken link, disconnected, circuit, bicycle",
```
```suggestion
        "tags": "electrical, broken graph, off, disconnect, open, broken link, disconnected, circuit",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Array",
         "iconName": "array",
-        "tags": "array",
+        "tags": "ellipses, items, list, more, dots, expand, collapse, details",
```
```suggestion
        "tags": "data, type, list, ellipses, items, more, dots, details",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Boolean Array",
         "iconName": "array-boolean",
-        "tags": "array, boolean",
+        "tags": "switch input, report, erase, half, snapshot",
```
```suggestion
        "tags": "data, type, list, toggle, true, false",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -1745,210 +1745,210 @@
         "displayName": "Segmented control",
         "iconName": "segmented-control",
-        "tags": "button, switch, option",
+        "tags": "button, switch, option, toggle, progress bar",
```
moved some of these from "array-boolean" to here, where they are more appropriate

```suggestion
        "tags": "button, switch, option, toggle, progress bar, boolean, erase, half",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Date Array",
         "iconName": "array-date",
-        "tags": "array, date",
+        "tags": "array, date, calendar array, calendars, battery, date array, list of dates, calendar, dates, array of dates, calendar in brackets",
```
```suggestion
        "tags": "data, type, list, date, time, datetime, calendar",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Numeric Array",
         "iconName": "array-numeric",
-        "tags": "array, numeric, number",
+        "tags": "numbers, digits, set, range, count, sort descending, integer",
```
```suggestion
        "tags": "data, type, list, numbers, digits, set, range, count, integer",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "String Array",
         "iconName": "array-string",
-        "tags": "array, string",
+        "tags": "quotes, quote, google hangouts",
```
```suggestion
        "tags": "data, type, list, quotes, quote",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Timestamp Array",
         "iconName": "array-timestamp",
-        "tags": "array, timestamp",
+        "tags": "clock, delay, timespan",
```
```suggestion
        "tags": "data, type, list, date, time, datetime, clock, delay, timespan",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Add Location",
         "iconName": "add-location",
-        "tags": "location, add,",
+        "tags": "location, new, plus, object object, node, circle",
```
```suggestion
        "tags": "location, new, plus, object, node, circle",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "One to one",
         "iconName": "one-to-one",
-        "tags": "object, relation, ontology",
+        "tags": "object, relation, ontology, link, connect, join, on, edge, nodes",
```
```suggestion
        "tags": "object, relation, ontology, link, connect, join, edge, nodes",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Geometry: Stadium",
         "iconName": "stadium-geometry",
-        "tags": "geometry, shape, race track, ellipse, discorectangle, obround",
+        "tags": "shape, race track, ellipse, round, oval, loop, track, home button",
```
actually these existing tags were intentional and precise terms for this shape:

```suggestion
        "tags": "shape, race track, ellipse, discorectangle, obround, round, oval, loop, track, home button",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Area of interest",
         "iconName": "area-of-interest",
-        "tags": "NAI, named area, location",
+        "tags": "marked, marker, location, region, geo, tag, locate, point, map",
```
why remove the existing tags? do you have context about them and why they were here in the first place?

```suggestion
        "tags": "NAI, named area, marked, marker, location, region, geo, tag, locate, point, map",
```

gabrieljroth replied:
My bad, I didn't realize they were existing ones!

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Buggy",
         "iconName": "buggy",
-        "tags": "vehicle, car, transportation",
+        "tags": "vehicle, car, transportation, empty, nothing",
```
```suggestion
        "tags": "vehicle, car, transportation, dune, offroad",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3075,644 +3075,644 @@
         "displayName": "Tank",
         "iconName": "tank",
-        "tags": "tank, automobile, vehicle, armored",
+        "tags": "automobile, vehicle, armored, military, tactic, army",
```
```suggestion
        "tags": "automobile, vehicle, armored, military, army",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Rain",
         "iconName": "rain",
-        "tags": "precipitation, storm, weather",
+        "tags": "precipitation, storm, weather, cloud, data",
```
```suggestion
        "tags": "precipitation, storm, weather, forecast, cloud",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Wind",
         "iconName": "wind",
-        "tags": "air, breath, draft, weather",
+        "tags": "air, breath, draft, weather, blowing, current",
```
```suggestion
        "tags": "air, breath, draft, weather, forecast, blowing, current",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Lightning",
         "iconName": "lightning",
-        "tags": "thunder, bolt, strike, weather",
+        "tags": "thunder, bolt, strike, weather, quick, flash, immediate, electricity, power, high voltage, spark",
```
```suggestion
        "tags": "thunder, bolt, strike, weather, forecast, quick, flash, immediate, electricity, power, high voltage, spark",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Hurricane",
         "iconName": "hurricane",
-        "tags": "storm, cyclone, typhoon, tropical storm",
+        "tags": "storm, cyclone, typhoon, tropical storm, swirl, fast, razor, saw, spin, whirl pool",
```
```suggestion
        "tags": "storm, cyclone, typhoon, tropical storm, weather, forecast, swirl, fast, razor, saw, spin, whirl pool",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Target",
         "iconName": "target",
-        "tags": "location, pinpoint, scope, center",
+        "tags": "location, pinpoint, scope, center, add, new, plus",
```
the "add" icon is much better suited for "add, new, plus"

```suggestion
        "tags": "location, pinpoint, scope, center",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3726,84 +3726,84 @@
         "displayName": "Small square",
         "iconName": "small-square",
-        "tags": "square",
+        "tags": "square, pixel",
```
```suggestion
        "tags": "square, pixel, outline, border",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3817,84 +3817,84 @@
         "displayName": "Symbol: rectangle",
         "iconName": "symbol-rectangle",
-        "tags": "symbol, rectangle, filled shape",
+        "tags": "filled shape, grey, box",
```
```suggestion
        "tags": "filled shape, box",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3817,84 +3817,84 @@
         "displayName": "Open application",
         "iconName": "open-application",
-        "tags": "open, application, new-tab, external",
+        "tags": "new tab, external, click website, link to page, export to pdf, share, open in pop-up, upload, data, spreadsheet, screen",
```
```suggestion
        "tags": "new tab, external, click website, link to page, export to pdf, share, open in pop-up, upload, data, window, screen",
```

adidahiya commented on packages/icons/icons.json:
```diff
@@ -3817,84 +3817,84 @@
         "displayName": "Floating point array",
         "iconName": "array-floating-point",
-        "tags": "number, float, double, decimal, array",
+        "tags": "number, float, double, decimal, array, input, format, versioning, number, tag",
```
```suggestion
        "tags": "data, type, list, number, numeric, float, double, decimal, format, version",
```

adidahiya reviewed (changes requested):
reviewed from line 2100 to the end

gabrieljroth committed:
Apply suggestions from code review

Left a few comments but accepted almost all changes!

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

gabrieljroth reviewed (commented):
Accepted almost all suggestions, left a few comments.

gabrieljroth committed:
Update packages/icons/icons.json

Co-authored-by: Adi Dahiya <adi.dahiya14@gmail.com>

adidahiya committed:
Apply suggestions from code review
